src/streamnet.cpp

```diff
@@ -337,14 +337,14 @@ int reachshape(long *cnet,float *lengthd, float *elev, float *area, double *poin
 
     //creating geometry using OGR
 
-	geometry = OGR_G_CreateGeometry( wkbMultiLineString );
+	//geometry = OGR_G_CreateGeometry( wkbMultiLineString );// want Linestring
```

Comment:
I would prefer to remove these commented code. If the commented code is needed for any reference then may be we can write it as a comment and not as commented code.